### ISSUE TITLE
Cannot move cursor in the Source Pane

### ISSUE THREAD

**Issue by abarthch** [rstudio/rstudio#4053]
### System details

    RStudio Edition : Server
    RStudio Version : 1.1.456
    OS Version      : iOS 12
    R Version       : 3.5.1

### Steps to reproduce the problem

Set the cursor in the Source Pane and try to scroll (either by hand/touch or with the arrow keys on the keyboard). I can set the cursor within the current section of the file, but it just doesn't move up or down. The only way I found to navigate through a text file in the Source Pane is by search feature.

### Describe the problem in detail

This problem occurs only on iOS, either with Safari or Chrome browser, on macOS and Safari it works fine. It also only occurs in the Source Pane, because in the Console I can move the cursor up/down and left/right by touch or keyboard.

I understand that RStudio Server probably isn't made for iOS specifically, but other than the scrolling feature everything works great so far. I can even import data over the iOS Dropbox import feature. This is just amazing, to be able to run R on an iPad and very convenient for learning in college.

![screenshot](https://user-images.githubusercontent.com/3002949/50148351-98a99780-02b8-11e9-89f2-7ed99263b462.jpg)

**Comment by jmcphers** [comment]
Good news, we've already fixed this in RStudio 1.2. See https://github.com/rstudio/rstudio/issues/2186 for details.

You can get a preview build of 1.2 here: https://www.rstudio.com/products/rstudio/download/preview/

There are still a number of known issues with RStudio Server on iOS, however: https://github.com/rstudio/rstudio/labels/ios

**Comment by AGeographer** [comment]
Hi,
I am experiencing not being able to move the cursor with the arrow keys in the Source pane in RStudio after a while for years. Never found out how to replicate it until now.
```
RStudio edition: Desktop
RStudio version: 1.2.5033
OS version:      Windows 10 LTSC
R version:       3.6.3
```
Moving the cursor with arrows works fine until I switch my keyboard language (`Windows + Space`) - I work in three different languages and frequently switch back and forth, esp. when making ggplot graphs. Thereafter the arrow keys do not show any reaction. And the `Delete` key then reacts like the `Backspace` key until I select another software that is open in a different window and switch back to RStudio.
It is really annoying, however, never found a way to replicate it until now. Hope this helps to find the bug and get rid of it finally.
Thank you for an otherwise excellent GUI!
